src/Bicep.LangServer/Snippets/Templates/res-template-spec.bicep

```diff
@@ -0,0 +1,9 @@
+//Template spec

```

Comment:
Please add a space between // and Template

Reply:
done!